src/main/resources/db/migration/V2_0__add_column_varselid_to_kandidat.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE SEN_OPPFOLGING_KANDIDAT
+    ADD COLUMN varsel_id CHAR(36) DEFAULT NULL;
+
+ALTER TABLE SEN_OPPFOLGING_KANDIDAT
+    ALTER COLUMN varsel_at DROP NOT NULL;
```

Comment:
LItt usikker på om vi burde sette varsel_at til null for eksisterende rader

Reply:
Synes vi burde det. Vi vet jo egentlig ikke når disse fikk varsel, så blir feil å la det være tidspunktet de svarte.